### ISSUE TITLE
Autocomplete on Author Submission Committee Members page (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#97]
When Jaws is used, autocomplete does not work as expected.  Issues include:

- Email field is not populated when an autocomplete item is selected. - adding aria tags to this item may fix this issue

- Only the first item on the drop-down list is read - the list contains aria tags and each list item has the same styling, tags, etc.  Not sure what is causing the behavior.  Try updating jquery-ui autocomplete?

**Comment by cam156** [comment]
https://api.jqueryui.com/autocomplete/ possibly try the change event instead of the select event

**Comment by cam156** [comment]
The fix in #113 made the autocomplete work almost all of the time.  Was able to confuse the change method by doing a bunch of changes and picking and then adding text and backing it out, which is similar to Michelle did to get the item to stop working.

**Comment by cam156** [comment]
Here are the instructions for getting the autocomplete to not fill in:
Select an item
Add a character
Remove a character
tab to next item
